docs/pages/versions/unversioned/distribution/uploading-apps.md

```diff
@@ -66,16 +66,14 @@ To upload your iOS app to TestFlight, run `expo upload:ios`. You can set followi
 
 ### Manually uploading your app
 
-In order to see your app on Testflight, you will first need to submit your .IPA file to Apple using Application loader. In order to do this, there are a few prequisite steps which you may not have followed previously if this is your first app submission to Apple:
+In order to see your app on Testflight, you will first need to submit your .IPA file to Apple using **Transporter** (previously known as Application Loader), available on the App Store ([link](https://apps.apple.com/app/transporter/id1450874784)). In order to do this:
 
-1. Make sure you have logged into iTunes connect at least once with your Apple ID and accepted the terms.
```

Comment:
Does this no longer apply? Not sure how to test that without creating a new Apple ID & dev account

Reply:
Good call, I only skimmed through the old instructions, didn't catching that it's actually an important one. (I hope Transporter would catch that and show some explanatory message, but let's not depend on that one).